scripts/models.py

```diff
@@ -0,0 +1,111 @@
+"""
+Behavior of our game work
+"""
+import random
+
+from ascii_img.choice_of_hero import choice_of_hero
+from ascii_img.game_end import game_over
+from ascii_img.thief_win import thief_win
+from ascii_img.warrior_win import warrior_win
+from ascii_img.wizard_win import wizard_win
+#from exceptions import EnemyDown
+#from exceptions import GameOver
+
+
+class Enemy:
+    def __init__(self, level):
+        self.level = level
+
+    @staticmethod
+    def select_attack():
+        return random.choice([1, 2, 3])
+
+    def decrease_lives(self):
+        self.level -= 1
+        if self.level == 0:
+            #raise EnemyDown()
+            print("Enemy Down!")
+        return self.level
+
+
+class Player:
+    def __init__(self, name, lives=5, score=0):  # , allowed_attacks):
```

Comment:
Move lives to settings and not to use common integers on object creation.
Do you really need to set a score as parameter?